site/en/docs/handbook/how-to/add-a-featured-card/index.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Add a Featured Card
```

Comment:
Can we call it more general, like "add a landing page" and include (not necessarily in this PR, but with time) all landing page components to it?